mmv1/third_party/terraform/services/servicenetworking/resource_service_networking_connection.go

```diff
@@ -118,7 +123,21 @@ func resourceServiceNetworkingConnectionCreate(d *schema.ResourceData, meta inte
 	}
 
 	if err := ServiceNetworkingOperationWaitTimeHW(config, op, "Create Service Networking Connection", userAgent, project, d.Timeout(schema.TimeoutCreate)); err != nil {
-		return err
+		if strings.Contains(err.Error(), "Cannot modify allocated ranges in CreateConnection. Please use UpdateConnection") && d.Get("update_on_creation_fail").(bool) {
```

Comment:
Can we shorten the string contains to potentially just the first sentence? This would lessen the chance that an arbitrary API change would break this feature.

How important overall do you think it is to have the string matching in the first place?

Reply:
Yeah, it is enough to have the first sentence.